ddp-workspace/projects/toolkit-prion/src/test.ts

```diff
@@ -1,6 +1,6 @@
 // This file is required by karma.conf.js and loads recursively all the .spec and framework files
 
-import 'core-js/es7/reflect';
+// import 'core-js/es7/reflect';
```

Comment:
Why do you comment this ?

Reply:
it throws an error. I wanted to remove it first, but then found out it's commented in other studies in polyfills.ts files. Toolkit doesn't have polyfills file. So I decided to follow the same pattern and commented it.